swan-lake/integration-tutorials/sending-a-message-to-a-service.md

```diff
@@ -93,17 +77,18 @@ Follow the instructions given in this section to develop the service.
         }
         ```
 
-6. Define an [`http:Client`](https://ballerina.io/learn/by-example/#http-client) object to send requests to the backend service.
+5. Define a [configurable variable](https://ballerina.io/learn/by-example/#configurability) for the URL of the backend service and an [`http:Client`](https://ballerina.io/learn/by-example/#http-client) object to send requests to the backend service.
 
-    ![Define the client](/learn/images/integration-tutorials/sending-a-message-to-a-service/define_client_endpoint.gif)
+    ![Define a configurable and a client](/learn/images/integration-tutorials/sending-a-message-to-a-service/define_a_configurable_and_a_client.gif)
 
     The generated code will be as follows.
 
     ```ballerina
+    configurable string healthcareBackend = "http://localhost:9090/healthcare";
     final http:Client queryDoctorEP = check new (url = healthcareBackend);
```

Comment:
Continuing the discussion re: https://github.com/ballerina-platform/ballerina-dev-website/pull/8057/files#r1374470440 here, I still think we should do it. Isn't it just a matter of selecting all (of the named arg), removing it, and just adding the variable reference?

Reply:
Found a way to do it properly. Please check